### ISSUE TITLE
Missing delete method in apps.js 

### ISSUE THREAD

**Issue by schawla3** [SmartThingsCommunity/smartapp-sdk-nodejs#49]
**Description**
There is no way to delete an app as Delete api request method is missing from client lib (lib/platform/app.js)

**Describe the solution you'd like**

```
delete(appId ) {
return this.st.client.request (`/apps/${id}`, 'DELETE')
}
```

**Expected behavior**
Deletes the app.

**Screenshots**
N/A

**Environment (please complete the following information):**
N/A

**Comment by erodewald** [comment]
Looks like it's missing alright: https://smartthings.developer.samsung.com/docs/api-ref/st-api.html#operation/deleteApp

App just requires `w:apps:*` as far as I know. I'm making a slow effort to document these things with JSDocs. Feel free to contribute. 

It would be ideal if there was a jsdoc comment attached to the function, as well. 

As an example of what I'm thinking:
```javascript
  /**
   * Delete an app.
   * @param {String} appId The appName or appId field of an app.
   * @returns {Request}
   */
  delete(appId) {
    // client call
  }
```

**Comment by erodewald** [comment]
Can I assign this to you @schawla3?

**Comment by schawla3** [comment]
yep fine by me :)

**Comment by erodewald** [comment]
Resolved by #51
